api/envoy/config/filter/network/http_connection_manager/v2/http_connection_manager.proto

```diff
@@ -175,6 +175,25 @@ message HttpConnectionManager {
   // option is not specified.
   google.protobuf.Duration drain_timeout = 12 [(gogoproto.stdduration) = true];
 
+  // The delayed close timeout is for downstream connections managed by the HTTP connection manager.
+  // It is defined as a grace period after connection close processing has been locally initiated
+  // during which Envoy will flush the write buffers for the connection and await the peer to close
+  // (i.e., a TCP FIN/RST is received by Envoy from the downstream connection).
+  //
+  // Delaying Envoy's connection close and giving the peer the opportunity to initate the close
+  // sequence mitigates a race condition that exists when downstream clients do not drain/process
+  // data in a connection's receive buffer after a remote close has been detected via a socket
+  // write(). This race leads to such clients failing to process the response code sent by Envoy,
+  // which could result in erroneous downstream processing.
+  //
+  // If the timeout triggers, Envoy will close the connection's socket.
+  //
+  // The default timeout is 1000 ms if this option is not specified.
+  //
+  // A value of 0 will completely disable delayed close processing, and the downstream connection's
+  // socket will be closed immediately after the write flush is completed.
+  google.protobuf.Duration delayed_close_timeout = 26 [(gogoproto.stdduration) = true];
```

Comment:
Sorry I forget this from before. Can you add this to the release notes also with a link to this field?

Reply:
Done.